src/components/data-exploration/heatmap/HeatmapMetadataTrackSettings.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useState, useRef } from 'react';
```

Comment:
I rewrote most of this component because it was a little bit hard to follow right now